src/ToDo.UI/Views/HomePage.xaml

```diff
@@ -63,7 +67,9 @@
 							 IsSettingsVisible="False"
 							 Style="{StaticResource HomeNavigationViewStyle}"
 							 CompactModeThresholdWidth="0"
-							 ExpandedModeThresholdWidth="0">
+							 ExpandedModeThresholdWidth="0"
+							 SelectedItem="{Binding SelectedList.Value}"
+							 SelectionChanged="SelectedListChanged">
```

Comment:
@dr1rrb Is there a way we can bind events to commands? perhaps using a behaviour?

Reply:
Behavior or listen for changes on the data-bound value